Taskfile.yml

```diff
@@ -5,15 +5,19 @@ version: '3'
 dotenv: ['.env']
 
 env:
-  OPSLEVEL_GO_PKG: "github.com/opslevel/opslevel-go/v2024"
+  vYEAR: "v2024"
+  OPSLEVEL_GO_PKG: "github.com/opslevel/opslevel-go/{{.vYEAR}}"
+  OPSLEVEL_JQ_PARSER_PKG: "github.com/opslevel/opslevel-jq-parser/{{.vYEAR}}"
+  OPSLEVEL_K8S_CONTROLLER_PKG: "github.com/opslevel/opslevel-k8s-controller/{{.vYEAR}}"
   SRC_DIR: "{{.TASKFILE_DIR}}/src"
 
 tasks:
   ci:
     desc: Workflow to run in CI
     cmds:
-      - task: workspace
-      - task: has-latest-opslevel-go
+      - task: has-latest-opslevel-dependencies
+      - task: tmp-ignore-go-work-file
+      - defer: { task: undo-ignore-go-work-files }
```

Comment:
@davidbloss shouldn't this defer statement be before `tmp-ignore-go-work-file`? In case `tmp-ignore-go-work-file` fails catastrophically causing the defer to not be queued?

Reply:
It's a pretty safe and small command. But sure I can shuffle these around